presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -263,11 +263,31 @@ Optionally, column metadata can be described in the same table description JSON
 ===================== ========= ============== =============================
 Field                 Required  Type           Description
 ===================== ========= ============== =============================
-``name``              optional  string         Column name of Elasticsearch field.
-``type``              optional  string         Column type of Elasticsearch `field`_.
-``jsonPath``          optional  string         Json path of Elasticsearch field.
-``jsonType``          optional  string         Json type of Elasticsearch field.
+``name``              required  string         Column name of Elasticsearch field.
+``type``              required  string         Column type of Elasticsearch field (see second column of `data type mapping`_).
```

Comment:
Is this supposed to be the Presto column type or the Elasticsearch column type?

Reply:
It's the Presto column type